.github/workflows/install-pkg.yml

```diff
@@ -39,25 +20,33 @@ jobs:
       # max-parallel: 6
       matrix:
         # PyTorch 1.5 is failing on Win and bolts requires torchvision>=0.5
-        os: [ubuntu-20.04]  # , windows-2019
+        os: [ubuntu-20.04, macOS-10.15 , windows-2019]  #
         python-version: [3.6, 3.7, 3.8]
 
     steps:
       - uses: actions/checkout@master
       - uses: actions/setup-python@v2
         with:
-          python-version: 3.7
+          python-version: ${{ matrix.python-version }}
 
       - name: Create package
         run: |
           # python setup.py check --metadata --strict
           python setup.py sdist
 
+      - name: Check package
+        run: |
+          pip install twine==1.13.0
+          twine check dist/*
+          python setup.py clean
+
       - name: Install package
         run: |
-          pip install virtualenv
-          virtualenv vEnv ; source vEnv/bin/activate
+          # pip install virtualenv
+          # virtualenv vEnv --system-site-packages
+          # source vEnv/bin/activate
           # pip install -r requirements.txt
-          pip install dist/*
+          pip install --find-links https://download.pytorch.org/whl/torch_stable.html dist/*
```

Comment:
Have you tried `pip install torch -f https://download.pytorch.org/who/torch_stable.html && pip install dist/*`?

Reply:
seems did not help, so the issue may be elsewhere...